### PR TITLE
{bio}[foss/2022a] cicero v1.3.8, Monocle3 v1.3.1 w/ R 4.2.1, R 4.2.1 Monocle3

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/m/Monocle3/Monocle3-1.3.1-foss-2022a-R-4.2.1.eb [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+name = 'Monocle3'
+version = '1.3.1'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://cole-trapnell-lab.github.io/monocle3/'
+description = """ An analysis toolkit for single-cell RNA-seq. """
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [('pkg-config', '0.29.2')]  # for textshaping
```
Comment (smoors) [inline easybuild/easyconfigs/m/Monocle3/Monocle3-1.3.1-foss-2022a-R-4.2.1.eb:12]: can we use `pkgconf` here instead?

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

**Comment by smoors** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**